charts/pgadmin4/templates/deployment.yaml

```diff
@@ -72,11 +72,12 @@ spec:
           securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
         {{- end }}
         {{- if .Values.command }}
-          command: {{ .Values.command }}
+          command:
+            {{- toYaml .Values.command | nindent 12 }}
         {{- end }}
         {{- if .Values.args }}
-          args: {{ .Values.args }}
-        {{- end }}
```

Comment:
this is still required, since the if statement remains

Reply:
I think you may be commenting on an outdated version. The if statement exists in the PR